motor_control/throttle/button_control.h

```diff
@@ -0,0 +1,20 @@
+#ifndef BUTTON_CONTROL
+#define BUTTON_CONTROL
+
+#include "base_throttle.h"
+#include <string>
+//#include <math.h>
+
+class ButtonThrottle: public Throttle{
+    public:
+    ButtonThrottle(const int &pin, int timeToFull = 5);
+    virtual int compute_motor_value() override;
```

Comment:
Not very sure this is necessary here. @ashvinnagarajan maybe you can weigh in?

Reply:
It doesnt need to be virtual since I can't imagine any class inheriting from the button_control class. Override is not necessary since it is already defined as virtual in the base class